integration/test/upgradetiller/upgrade_tiller_test.go

```diff
@@ -18,19 +19,55 @@ func TestUpgradeTiller(t *testing.T) {
 
 	var err error
 
-	// Install tiller with the latest image.
-	err = config.HelmClient.EnsureTillerInstalled(ctx)
+	labelSelector := "app=helm,name=tiller"
+	outdatedTillerImage := "gcr.io/kubernetes-helm:v2.7.2"
+	tillerNamespace := "giantswarm"
+
+	var helmClient *helmclient.Client
+	{
+		c := helmclient.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+
+			RestConfig:      config.RestConfig,
+			TillerImage:     outdatedTillerImage,
```

Comment:
I personally would prefer to create the `helmClient` with the default and change the value just before installing outdated tiller and then revert it. I think the intention would be clearer for the test reader that way. It's easy to miss this detail here IMO.

Reply:
@kopiczko I looked at this but the tiller image is internal. So I don't see how it can be changed on the fly. I've added a comment about using the outdated image first to make that clearer.

Reply:
You are right. I made some proposal: https://github.com/giantswarm/helmclient/pull/73

Reply:
Feel free to merge it to your branch if you find it ok. We can also work on #73 before merging.